data/fields/maxheight.json

```diff
@@ -1,6 +1,6 @@
 {
     "key": "maxheight",
     "type": "roadheight",
-    "label": "Max Height",
+    "label": "Vertical Clearance",
```

Comment:
Though this term is officially used in the U.S., it may be less familiar to some users who see these signs and think of them as simply “height limits”. Fortunately, schema-builder lets you specify `terms` for this field that will show up when searching for the field in iD. It’s probably a good idea to add all the previous labels as terms to accommodate existing users’ muscle memory.